core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -111,14 +142,18 @@ angular.module('oppia').controller(
       };
 
       $scope.acceptAndReviewNext = function() {
+        $scope.finalCommitMessage = generateCommitMessage();
+        if ($scope.isTranslationUpdated) {
+          $scope.finalCommitMessage = $scope.finalCommitMessage + '-With Edits';
```

Comment:
Prefer " (With Edits)".

Also, I think we want to append something like " This suggestion was submitted with reviewer edits." in the review message and surface the review message for accepted suggestions as well.

Reply:
Done.